src/main/java/org/sagebionetworks/web/client/widget/verification/VerificationSubmissionWidget.java

```diff
@@ -425,22 +434,14 @@ public void suspendVerification() {
   }
 
   private void rejectSuspendVerification() {
-    RejectReasonWidget promptModal = ginInjector.getRejectReasonWidget();
```

Comment:
Can RejectReasonWidget be deleted?

Reply:
It's still used by some DataAccessRequest widgets, though those may also be deletable